server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSPerformSPOptionsCheckUtility.java

```diff
@@ -0,0 +1,282 @@
+                    put(CICSLexer.FILE, ErrorSeverity.WARNING);
+                    put(CICSLexer.IPCONN, ErrorSeverity.WARNING);
+                    put(CICSLexer.JOURNALNAME, ErrorSeverity.WARNING);
+                    put(CICSLexer.JOURNALNUM, ErrorSeverity.WARNING);
+                    put(CICSLexer.JVMPROGRAM, ErrorSeverity.WARNING);
+                    put(CICSLexer.LIBRARY, ErrorSeverity.WARNING);
+                    put(CICSLexer.LSRPOOL, ErrorSeverity.WARNING);
+                    put(CICSLexer.MONITOR, ErrorSeverity.WARNING);
+                    put(CICSLexer.MQCONN, ErrorSeverity.WARNING);
+                    put(CICSLexer.MQMONITOR, ErrorSeverity.WARNING);
+                    put(CICSLexer.NODEJSAPP, ErrorSeverity.WARNING);
+                    put(CICSLexer.POLICY, ErrorSeverity.WARNING);
+                    put(CICSLexer.PROGAUTO, ErrorSeverity.WARNING);
+                    put(CICSLexer.PROGRAM, ErrorSeverity.WARNING);
+                    put(CICSLexer.PROGRAMDEF, ErrorSeverity.WARNING);
+                    put(CICSLexer.RECOVERY, ErrorSeverity.WARNING);
+                    put(CICSLexer.STATS, ErrorSeverity.WARNING);
+                    put(CICSLexer.STORAGE, ErrorSeverity.WARNING);
+                    put(CICSLexer.STREAMNAME, ErrorSeverity.WARNING);
+                    put(CICSLexer.SYSDUMP, ErrorSeverity.WARNING);
+                    put(CICSLexer.TABLEMGR, ErrorSeverity.WARNING);
+                    put(CICSLexer.TCPIP, ErrorSeverity.WARNING);
+                    put(CICSLexer.TCPIPSERVICE, ErrorSeverity.WARNING);
+                    put(CICSLexer.TDQUEUE, ErrorSeverity.WARNING);
+                    put(CICSLexer.TERMINAL, ErrorSeverity.WARNING);
+                    put(CICSLexer.TRANCLASS, ErrorSeverity.WARNING);
+                    put(CICSLexer.TCLASS, ErrorSeverity.WARNING);
+                    put(CICSLexer.TRANDUMP, ErrorSeverity.WARNING);
+                    put(CICSLexer.TRANSACTION, ErrorSeverity.WARNING);
+                    put(CICSLexer.TSQUEUE, ErrorSeverity.WARNING);
+                    put(CICSLexer.URIMAP, ErrorSeverity.WARNING);
+                    put(CICSLexer.USER, ErrorSeverity.WARNING);
+                    put(CICSLexer.VTAM, ErrorSeverity.WARNING);
+                    put(CICSLexer.WEBSERVICE, ErrorSeverity.WARNING);
+                    put(CICSLexer.XMLTRANSFORM, ErrorSeverity.WARNING);
+                }
+            };
+    public CICSPerformSPOptionsCheckUtility(
+            DialectProcessingContext context, List<SyntaxError> errors) {
+        super(context, errors, DUPLICATE_CHECK_OPTIONS);
+    }
+
+    /**
+     * Entrypoint to check CICS Perform System Command rule options
+     *
+     * @param ctx ParserRuleContext subclass containing options
+     * @param <E> A subclass of ParserRuleContext
+     */
+    public <E extends ParserRuleContext> void checkOptions(E ctx) {
+        switch (ctx.getRuleIndex()) {
```

Comment:
reset is not handle here nor in the grammar.

Reply:
Assuming RESETTIME is referred : since command does not have an option, what should be handled ?